src/components/loginPage/loginPage.js

```diff
@@ -45,6 +46,12 @@ const LoginPage = () => {
             <div className="logo-main">
                 <h1 className="logo">LOGO</h1>
             </div>
+
+            <div className="img-wrap">
+                <div className="img-wraped">
+                    <img src={bgimg} alt="bg"/>
+                </div>
+            </div>
```

Comment:
Hawing img element wrapped in 2 divs is redundant. If we'll need to have them wrapped later we'll do it later
 ```suggestion
            <div className="img-wraper">
                    <img src={bgimg} className="img-register-bg" alt="bg"/>
            </div>
```